FWCore/Concurrency/interface/WaitingTaskList.h

```diff
@@ -164,12 +164,12 @@ namespace edm {
     WaitNode* createNode(WaitingTask* iTask);
 
     // ---------- member data --------------------------------
-    std::atomic<WaitNode*> m_head;
+    std::atomic<WaitNode*>      m_head;
     std::unique_ptr<WaitNode[]> m_nodeCache;
     CMS_THREAD_GUARD(m_waiting) std::exception_ptr m_exceptionPtr;
-    unsigned int m_nodeCacheSize;
+    unsigned int              m_nodeCacheSize;
```

Comment:
Here the alignment is inconsistent. Maybe `clang-format` gets confused by `CMS_THREAD_GUARD`?

Reply:
Right - it must not thing of CMS_THREAD_GUARD(blah..) as a declaration